dcompiler.dd

```diff
@@ -472,33 +472,27 @@ $(WINDOWS
         Adjusts the compiler to implement D as a $(LINK2 spec/betterc.html, better C):
 
 	$(UL
+	$(LI Predefines `D_betterC` $(LINK2 spec/version.html#predefined-versions, version).)
 	$(LI $(LINK2 spec/expression.html#AssertExpression, Assert Expressions), when they fail,
 	call the C runtime library assert failure function
 	rather than a function in the D runtime.)
-
 	$(LI $(LINK2 https://dlang.org/spec/arrays.html#bounds, Array overflows)
 	call the C runtime library assert failure function
 	rather than a function in the D runtime.)
-
 	$(LI $(LINK2 spec/statement.html#final-switch-statement/, Final switch errors)
 	call the C runtime library assert failure function
 	rather than a function in the D runtime.)
-
 	$(LI Does not automatically link with phobos runtime library.)
-
 	$(UNIX
 	$(LI Does not generate Dwarf `eh_frame` with full unwinding information, i.e. exception tables
 	are not inserted into `eh_frame`.)
 	)
-
 	$(LI Module constructors and destructors are not generated meaning that
 	$(LINK2 spec/class.html#StaticConstructor, static) and
 	$(LINK2 spec/class.html#SharedStaticConstructor, shared static constructors) and
 	$(LINK2 spec/class.html#StaticDestructor, destructors)
 	will not get called.)
-
 	$(LI `ModuleInfo` is not generated.)
-
 	$(LI $(LINK2 http://dlang.org/phobos/object.html#.TypeInfo, `TypeInfo`)
 	instances will not be generated for structs.)
```

Comment:
FWIW we seem to have to lists of effects of `-betterC`:

https://dlang.org/dmd.html#switch-betterC
https://dlang.org/spec/betterc.html#consequences

Might be worthwhile to merge them at some point.